crates/celo-revm/src/api/celo_block_env.rs

```diff
@@ -3,11 +3,17 @@ use op_revm::L1BlockInfo;
 use revm::primitives::Log;
 use std::vec::Vec;
 
+#[derive(Debug, Clone, Default)]
+pub struct Cip64Info {
+    pub actual_intrinsic_gas_used: u64,
```

Comment:
A comment what this includes would be nice. Is it only debit + credit or does it have some fixed amount added to it (after all the fee currency intrinsic gas value is only a part of the total intrinsic gas)? Or call this `debit_credit_gas_used`. I would assume that all `gas_used` values are "actual" values.